src/index.html

```diff
@@ -1,13 +1,358 @@
+  </head>
+
+  <body class="page__content">
+    <header class="header">
+      <a class="logo-link"
+         href="#home">
+        <img src="images/logo.png"
+             alt="Moyo logo">
+      </a>
+      <nav class="main-nav">
+        <ul class="main-nav-list">
+          <li class="main-nav-item">
+            <a class="main-nav-item__link is-active"
+               href="#">Apple</a>
+          </li>
+          <li class="main-nav-item">
+            <a class="main-nav-item__link"
+               href="#">Samsung</a>
+          </li>
+          <li class="main-nav-item">
+            <a class="main-nav-item__link"
+               href="#">Smartphones</a>
+          </li>
+          <li class="main-nav-item">
+            <a data-qa="hover"
+               class="main-nav-item__link"
+               href="#">Laptops & Computers</a>
+          </li>
+          <li class="main-nav-item">
+            <a class="main-nav-item__link"
+               href="#">Gadgets</a>
+          </li>
+          <li class="main-nav-item">
+            <a class="main-nav-item__link"
+               href="#">Tablets</a>
+          </li>
+          <li class="main-nav-item">
+            <a class="main-nav-item__link"
+               href="#">photo</a>
+          </li>
+          <li class="main-nav-item">
+            <a class="main-nav-item__link"
+               href="#">Video</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <div class="product-container">
+      <section class="product-container__cards">
+
+        <div class="product-container__card"
+             data-qa="card">
+          <img class="product-container__card-pic"
+               src="./images/imac.jpeg"
+               alt="APPLE A1419 iMac 27 Retina 5K Monoblock (MNED2UA/A)">
+          <h4 class="product-container__card-title">
+            APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+          </h4>
+          <p class="product-container__card-product-code">Product code: 195434</p>
+          <div class="stars">
+            <div class="stars__container">
+              <div class="stars stars--4">
+                <span class="stars__star"></span>
+                <span class="stars__star"></span>
+                <span class="stars__star"></span>
+                <span class="stars__star"></span>
+                <span class="stars__star"></span>
+                <div class="stars__reviews">Reviews: 5</div>
+              </div>
+            </div>
+          </div>
+          <div class="product-price-container">
+            <div class="product-price-container__pricetag">
+              <span class="product-price-container__price">Price:</span>
+              <span class="product-price-container__amount">$2,199</span>
+            </div>
+          </div>
+          <div class="button-container">
+            <button class="button-container__buy-button">
+              <a class="button-container__buy-button-link"
+                 data-qa="hover"
```

Comment:
checklist point says 
`add data-qa="card-hover" (not just hover) to the link Buy inside the first card`